tests/libmigration/migration.go

```diff
@@ -414,15 +414,13 @@ func ConfirmVMIPostMigrationAborted(vmi *v1.VirtualMachineInstance, migrationUID
 }
 
 func RunMigrationAndExpectFailure(migration *v1.VirtualMachineInstanceMigration, timeout int) string {
-	var err error
+
 	virtClient := kubevirt.Client()
 	By("Starting a Migration")
-	Eventually(func() error {
-		migration, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(context.Background(), migration, metav1.CreateOptions{})
-		return err
-	}, timeout, 1*time.Second).ShouldNot(HaveOccurred())
-	By("Waiting until the Migration Completes")
+	_, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(context.Background(), migration, metav1.CreateOptions{})
+	Expect(err).ToNot(HaveOccurred())
 
+	By("Waiting until the Migration Completes")
 	uid := ""
 	Eventually(func() v1.VirtualMachineInstanceMigrationPhase {
 		migration, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Get(context.Background(), migration.Name, metav1.GetOptions{})
```

Comment:
The following approach will solve this issue.

```suggestion
	createdMigration, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(context.Background(), migration, metav1.CreateOptions{})
	Expect(err).ToNot(HaveOccurred())

	By("Waiting until the Migration Completes")
	uid := ""
	Eventually(func() v1.VirtualMachineInstanceMigrationPhase {
		migration, err := virtClient.VirtualMachineInstanceMigration(createdMigration.Namespace).Get(context.Background(), createdMigration.Name, metav1.GetOptions{})
```

Reply:
shit, that's my bad i missed that our `libmigration.New(vmi.Name, vmi.Namespace)`  method initialize the migration with a `GenerateName` prefix, so when it's created its gets a full name, so we must store the returned value.

Thanks!!